### DISCUSSION TITLE
How to check the information being sent as context

### DISCUSSION THREAD

**Discussion by silverskyvicto** [community/community#138158] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

In GitHub Copilot Business, is there a way for an administrator to see from the admin screen what information is being submitted as context?

### Background Information

GitHub Copilot Business allows you to set up content exclusions, but if you do not set them up correctly, your data may be sent.
In such cases, if there is a way for administrators to see what information is being sent, they can determine what action is necessary.

**Comment by rioyuda02** [comment, marked as answer]
Hi,

GitHub Copilot Business does not provide a way for administrators to directly view the specific information that is being submitted as context to GitHub Copilot from the admin screen. While administrators can set up content filters and exclusions to control what data Copilot has access to, they cannot inspect the actual data Copilot uses in real-time or after-the-fact from the admin dashboard.

Here are the relevant features that GitHub Copilot Business provides regarding privacy and data control:

Content Exclusions: Administrators can configure content filtering to prevent sensitive code, such as secret keys, from being sent to GitHub Copilot servers. However, if these exclusions are not set correctly, the possibility remains that sensitive information could be transmitted.

Telemetry Control: GitHub offers an option to disable telemetry data collection, but even with this control, it does not provide detailed logs of what is sent.

Policy Enforcement: Admins can enforce policies regarding how GitHub Copilot interacts with repositories and what type of data is used, but they do not have visibility into the actual data submitted during code suggestions.

Recommended Action
If content filtering is not set correctly and you're concerned about sensitive data being sent, you should ensure the content exclusion filters are properly configured. Additionally, you could enforce organization-wide best practices on handling sensitive data within repositories.

If you need detailed insights, your best approach would be to work directly with GitHub support or implement manual code reviews to inspect sensitive areas that might be exposed through Copilot interactions.

Hopefully answered.

**Reply by silverskyvicto** [reply]
@rioyuda02 

Thank you for answer.
GitHub Copilot Enterprise is the same, right?
